test/Java8andUp/src_90/org/openj9/test/java/lang/Test_System.java

```diff
@@ -144,9 +144,11 @@ public void test_currentTimeMillis() {
 	 */
 	@Test
 	public void test_getProperties() {
+		String versionStringPrefix = System.getProperty("JavaVersion");
```

Comment:
Using JAVA_VERSION as expected value to verify getProperty("java.version") is a little bit risky to me as user exports this JAVA_VERSION value. If user sets it wrong, then our test will fail.